draft-irtf-cfrg-vdaf.md

```diff
@@ -379,54 +379,264 @@ exception that Clients submitting measurements may be anonymous.
 
 # Definition of DAFs {#daf}
 
-> TODO Issue#20
+By way of a gentle introduction to VDAFs, this section describes a simpler class
+of schemes called Distributed Aggregation Functions (DAFs). Unlike VDAFs, DAFs
+do not provide verifiability of the computation. Clients must therefore be
+trusted to compute their input shares correctly. Because of this fact, the use
+of a DAF is NOT RECOMMENDED for most applications. See {{security}} for
+additional discussion.
+
+A DAF scheme is used to compute a particular "aggregation function" over a set
+of measurements generated by Clients. An aggregation function may include an
+optional "aggregation parameter", which is selected by the Collector and
+disseminated among the Aggregators. The semantics of this parameter is specific
+to the aggregation function, but in general it is used to represent the set of
+"queries" that can be made on the measurement set
+
+Execution of a DAF has four distinct stages:
+
+* Sharding - Each Client generates input shares from its measurement and
+  distributes them among the Aggregators.
+* Preparation - Each Aggregator converts each input share into an output share
+  compatible with the aggregation function. This computation involves the
+  aggregation parameter. In general, each aggregation parameter may result in a
+  different an output share.
+* Aggregation - Each Aggregator combines a sequence of output shares into its
+  aggregate share and sends the aggregate share to the Collector.
+* Unsharding - The Collector combines the aggregate shares into the aggregate
+  result.
+
+Sharding and Preparation are done once per measurement input.  Aggregation and
```

Comment:
I think this sentence makes sense if we read "input" as a verb and not a noun, but it's a bit confusing given that those two words have specific meanings throughout VDAF and DAP. Maybe rephrase to only use one or the other word?